frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -1,15 +1,11 @@
 import * as React from 'react';
 import * as _ from 'lodash';
-import { LoadingInline, pluralize } from '@console/internal/components/utils';
 import { RedExclamationCircleIcon } from '@console/shared';
-
-const pluralizeWithNotation = (i: number, suffix: string, singular: string) =>
-  `${i}${suffix} ${i === 1 ? singular : `${singular}s`}`;
+import { LoadingInline, pluralize, humanizeNumber } from '@console/internal/components/utils';
```

Comment:
Imported the `pluralize` but not using it.

Reply:
Its actually being used  for titles in Buckets row :)

Reply:
ok @afreen23